components/NavigationDrawer/examples/BottomDrawerExpandFullscreenExample.swift

```diff
@@ -17,7 +17,7 @@ import MaterialComponents.MaterialBottomAppBar
 import MaterialComponents.MaterialBottomAppBar_ColorThemer
 import MaterialComponents.MaterialColorScheme
 import MaterialComponents.MaterialNavigationDrawer
-import MaterialComponents.MaterialNavigationDrawer_ColorThemer
```

Comment:
These changes appear unrelated, perhaps they should be split out into a separate PR

Reply:
https://github.com/material-components/material-components-ios/pull/6171